tests/core/structure/cellReferences.test.ts

```diff
@@ -107,17 +106,12 @@ describe("Cell references", () => {
   });
 
   it("should create an empty cell with styling", () => {
-    const b2 = sheet.getCellInfoAt(1, 1)!;
-    sheet.setCellStyle(
-      b2.position!.columnKey!,
-      b2.position!.rowKey!,
-      new CellStyle(new Map([["width", "50px"]])),
-    );
+    sheet.setCellCss(1, 1, new Map<string, string>([["width", "50px;"]]));
 
     sheet.setCellAt(1, 1, "");
 
     // Clearing the style should result in the cell being deleted.
-    sheet.setCellStyle(b2!.position.columnKey!, b2!.position.rowKey!, null);
+    sheet.setCellCss(1, 1, new Map<string, string>());
     expect(sheet.getCellInfoAt(1, 1)).toBeNull();
   });
```

Comment:
This test is failing. maybe I did something wrong something. I am a bit tired as well, It would be nice if someone can get me insights on why this maybe happening.